src/components/SidePanes/CatalogInclusionPane.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Form, Spinner } from '@edx/paragon';
+import {
+  PUBLISHED, REVIEW_BY_INTERNAL, REVIEWED, UNPUBLISHED,
+} from '../../data/constants';
+
+import DiscoveryDataApiService from '../../data/services/DiscoveryDataApiService';
+import Pane from './Pane';
+
+const CatalogInclusionPane = ({ courseUuid, subInclusion, draftStatuses }) => {
+  const [inclusion, setInclusion] = useState(subInclusion);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(false);
+
+  const toggleInclusion = async (e) => {
+    e.preventDefault();
+    setIsLoading(true);
+    let draft = null;
+    if (draftStatuses.includes(PUBLISHED)) {
+      draft = false;
+    }
+    if (draftStatuses.includes(UNPUBLISHED)) {
+      draft = true;
+    }
+    if (draftStatuses[0] === REVIEWED || draftStatuses.includes(REVIEW_BY_INTERNAL)) {
+      setIsLoading(false);
+      setError(true);
+    }
+
+    if (draft !== null) {
+      const data = {
+        uuid: courseUuid,
+        draft,
+        enterprise_subscription_inclusion: !(inclusion),
+      };
+      try {
+        await DiscoveryDataApiService.editCourse(data);
+        setInclusion(!(inclusion));
+        setIsLoading(false);
```

Comment:
if the API call is successful, setError(false) should be called. Otherwise, if prev API call had an error, the message will remain there despite the next call being successful.